main/src/com/google/refine/importing/ImportingUtilities.java

```diff
@@ -309,11 +310,18 @@ public void update(long bytesRead, long contentLength, int itemCount) {
                         CloseableHttpResponse response = httpclient.execute(httpGet);
 
                         try {
-                            response.getStatusLine();
+                            StatusLine status = response.getStatusLine();
                             HttpEntity entity = response.getEntity();
                             if (entity == null) {
                                 throw new Exception("No content found in " + url.toString());
                             }
+                            int statusCode = response.getStatusLine().getStatusCode();
+                            if (statusCode / 100 != 2) {
```

Comment:
The standard way to do this is to consider anything greater or equal to 400 to be an error. 
```suggestion
                            if (statusCode >= 400) {
```